config/initializers/secure_headers.rb

```diff
@@ -9,6 +10,8 @@
   form_action << %w[localhost:3000] if Rails.env.development?
   connect_src = ["'self'"]
   connect_src << %w[ws://localhost:3036 http://localhost:3036] if Rails.env.development?
+  script_src = ["'self'", '*.newrelic.com', '*.nr-data.net']
+  script_src << "'unsafe-eval'" if Rails.env.development? || Rails.env.test?
```

Comment:
As mentioned in Slack, if we're not using `eval-sourcemap` Webpack devtool, hopefully we can avoid `unsafe-eval` in the CSP.

Reply:
good catch